script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -1,7 +1,7 @@
 [sha:d2f000479b76e451252cd3349ddf074bd8159bd8]{
 -- Page 10012, Offset 25003.
 -- 午後２時を過ぎ、遅めの昼食を摂る事になった。
-It's past 2:00PM now, so we decide to have a late lunch.
+It's past 2 PM now, so we decide to have a late lunch.
```

Comment:
I thought these were supposed to be two o'clock? Even if I do personally prefer digits.